### DISCUSSION TITLE
Composite PKs - leading to errors [Aug 30]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3732] [Discord Threads]
> 🙋‍♂️ **MOV**  
#### did somebody have sometime similar problem?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1011153117699637358/unknown.png)  
![image1](https://cdn.discordapp.com/attachments/661905455894888493/1011153118081335296/try_to_save.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **Amin Debabeche**  
#### I had the same issue !  
#  
> 🙋‍♂️ **MOV**  
#### I checked and it is problem from windows and Mac too😵‍💫  
#  
> 🙋‍♂️ **Amin Debabeche**  
#### Here we go with another type of error while importing an Excel file. Some help would be cool  
![image0](https://cdn.discordapp.com/attachments/1011153119285096448/1011209347294040115/Capture_decran_2022-08-22_a_11.44.00.png)  
#  
> 💁 **o1lab**  
#### Looks like in the first image - there are composite pks  
#  
> 💁 **o1lab**  
#### and in second one you are creating another primary key with name ID  
#  
> 💁 **o1lab**  
#### when table is created - we automatically create PK for you  
#  
> 💁 **o1lab**  
#### is that what you intended to do so ? create composite primary keys  
#  
> 💁 **o1lab**  
#### there is a bug open for composite pks  
#  
> 💁 **o1lab**  
#### Amin, would need more logs and pls file an issue   
####    
#### if you are importing excel file - you will have to map the fields correctly  
#  
> 🙋‍♂️ **Amin Debabeche**  
#### maps the field correctly with the data type you mean ?  
#  
> 💁 **o1lab**  
#### yes  
#  
> 💁 **o1lab**  
#### pls note - upload is in alpha as well   
####    
#### upload is good to upload content to tables instead of lookups, linktoAnotherRecords (other table fields)  
#  
> 🙋‍♂️ **Amin Debabeche**  
#### Is there a possibility to see an extensive log to know where my data is not mapped correctly ?  
#  
> 🙋‍♂️ **MOV**  
#### thank you!   
#### I deleted one primary key column and it started working  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1011153119285096448)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
